napari/utils/action_manager.py

```diff
@@ -99,12 +100,30 @@ def _validate_action_name(self, name):
                 )
             )
 
+    def _is_repeatable(self, name) -> bool:
+        """
+
```

Comment:
Nit: remove the whitespace. Or maybe better, just condense the whole docstring into a one or two line summary as this is a private/helper function that is relatively simple.

Actually, is this used? I did a quick search for it on the GitHub file diff and did not find it. If that's correct, just delete this as the public equivalent is just as readable as calling this method (i.e. `self.actions[name].repeatable` vs.` self._is_repeatable(name)`).

Reply:
Thanks for the nit-picks, I'll change them right away.

Good call on this private function, defined it during development and failed to realize it's not needed in the current implementation.